waku/v2/node/storage/message/waku_message_store.nim

```diff
@@ -12,18 +13,59 @@ import
 
 export sqlite
 
+logScope:
+  topics = "wakuMessageStore"
+
 const TABLE_TITLE = "Message"
+const MaxStoreOverflow = 1.3
```

Comment:
Although it would never happen probably, it is prudent to add a comment that this value must be larger than `1`, otherwise the math at line 98, i.e., 
> let deleteWindow = int(float(storeMaxLoad - storeCapacity) / 2)
will yield a negative value.

Reply:
I will do this. Thanks :).